packages/office-ui-fabric-react/src/utilities/dateMath/DateMath.ts

```diff
@@ -200,18 +198,119 @@ function getDatePartHashValue(date: Date) {
   /* tslint:enable:no-bitwise */
 }
 
+// Returns the week numbers for each week in a month.  Week numbers are 1 - 52 (53) in a year
```

Comment:
Add the exports before the private functions and follow the existing commenting conventions please

Reply:
#Resolved